docs/user_guide/network_explorer.md

```diff
@@ -31,6 +31,17 @@ By clicking on an arrow in the SLD you can navigate to another voltage level.
 
 Switches can also be clicked, causing their status in the network to change; Please note that, currently, this action does not trigger any computation on the network  (e.g., a LF is not   automatically run on the network).
 
+## Network map tab
+
+![network-explorer MAP tab](/_static/img/network_explorer_3.png)
+
+The 'Network map' displays the network's substations and lines on a map. The map is empty if the network does not contain geo data.
```

Comment:
The map is empty but centered and zoomed on latitude 0, longitude 0, which is strange. It would be better to have the map unzoomed to see the whole world. Or not to load the map viewer widget, replacing it with a text "No geodata found"? what do you think?